<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-CEESDAM" ignored="0" name="Test" tests="7" failures="1" timestamp="2022-11-29T10:11:38 EST" time="36.429" errors="0">
  <testcase name="logingTest" time="1.882" classname="com.H2K.TestCases.TC_Loging_001"/>
  <testcase name="logingTest" time="2.343" classname="com.H2K.TestCases.TC_Loging_001"/>
  <testcase name="logingTest" time="1.889" classname="com.H2K.TestCases.TC_Loging_001"/>
  <testcase name="logingTest" time="1.867" classname="com.H2K.TestCases.TC_Loging_001"/>
  <testcase name="logingTest" time="1.841" classname="com.H2K.TestCases.TC_Loging_001"/>
  <testcase name="validatetest" time="0.005" classname="com.H2K.TestCases.TC_Loging_001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.H2K.TestCases.TC_Loging_001.validatetest(TC_Loging_001.java:31)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validatetest -->
  <testcase name="landing" time="1.977" classname="com.H2K.TestCases.TC_Landing_002"/>
</testsuite> <!-- Test -->
